modules/oe_list_pages_open_vocabularies/tests/src/FunctionalJavascript/OpenVocabulariesFiltersTest.php

```diff
@@ -0,0 +1,388 @@
+    'oe_list_pages_link_list_source',
+    'open_vocabularies',
+    'node',
+    'emr',
+    'emr_node',
+    'search_api',
+    'search_api_db',
+  ];
+  /**
+   * The association to test.
+   *
+   * @var \Drupal\open_vocabularies\OpenVocabularyAssociationInterface
+   */
+  protected $association;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+    /** @var \Drupal\emr\EntityMetaRelationInstaller $installer */
+    $installer = \Drupal::service('emr.installer');
+    $installer->installEntityMetaTypeOnContentEntityType('oe_list_page', 'node', [
+      'oe_list_page',
+    ]);
+
+    // Create OpenVocabularies vocabularies.
+    $values = [
+      'id' => 'custom_vocabulary',
+      'label' => 'My amazing vocabulary',
+      'description' => $this->randomString(128),
+      'handler' => 'taxonomy',
+      'handler_settings' => [
+        'target_bundles' => [
+          'entity_test' => 'vocabulary_one',
+        ],
+      ],
+    ];
+
+    /** @var \Drupal\open_vocabularies\OpenVocabularyInterface $vocabulary */
+    $vocabulary = OpenVocabulary::create($values);
+    $vocabulary->save();
+
+    // Create association for content types.
+    $fields = [
+      'node.content_type_one.field_open_vocabularies',
+      'node.content_type_two.field_open_vocabularies',
+    ];
+    $values = [
+      'label' => 'My amazing association',
+      'name' => 'open_vocabulary',
+      'widget_type' => 'options_select',
+      'required' => TRUE,
+      'help_text' => 'Some text',
+      'predicate' => 'http://example.com/#name',
+      'cardinality' => 5,
+      'vocabulary' => 'custom_vocabulary',
+      'fields' => $fields,
+    ];
+
+    $this->container->get('kernel')->rebuildContainer();
+    /** @var \Drupal\open_vocabularies\OpenVocabularyAssociationInterface $association */
+    $association = OpenVocabularyAssociation::create($values);
+    $association->save();
+    $association_id = $association->id();
+    $this->association = $association;
+
+    // Create some taxonomy terms.
+    $values = [
+      'name' => 'yellow color',
+      'vid' => 'vocabulary_one',
+    ];
+    $term_1 = Term::create($values);
+    $term_1->save();
+
+    $values = [
+      'name' => 'green color',
+      'vid' => 'vocabulary_one',
+    ];
+    $term_2 = Term::create($values);
+    $term_2->save();
+
+    // Create some test nodes to index and search in.
+    $date = new DrupalDateTime('09-02-2021');
+    $values = [
+      'title' => 'one yellow fruit',
+      'type' => 'content_type_one',
+      'body' => 'this is a banana',
+      'status' => NodeInterface::PUBLISHED,
+      'field_select_one' => 'test1',
+      'field_open_vocabularies' => [
+        'target_id' => $term_1->id(),
+        'target_association_id' => $association_id,
+      ],
+      'created' => $date->getTimestamp(),
+    ];
+    $node = Node::create($values);
+    $node->save();
+    $values = [
+      'title' => 'another yellow fruit',
```

Comment:
Why not call this 'a green fruit' since we are tagging it with "green color"? makes it confusing after to read the tests.